applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh

```diff
@@ -4,8 +4,12 @@ cmake .. \
 -DSCOTCH_INCLUDE_DIR="/usr/include/scotch/"	            \
 -DSCOTCH_LIBRARY="libscotchmetis-5.1.so"	            \
 -DSCOTCHERR_LIBRARY="libscotcherr-5.1.so"	            \
--DCMAKE_CXX_FLAGS="-O3 -mavx2 -fPIC -fopenmp"               \
--DCMAKE_C_FLAGS="-O3 -mavx2 -fPIC -fopenmp"                 \
+-DCMAKE_CXX_FLAGS="-O3 -fPIC -fopenmp"               \
+-DCMAKE_C_FLAGS="-O3 -fPIC -fopenmp"                 \
```

Comment:
You should try to use at least a less modern set of SIMD instructions if AVX2 is not available in your processor.

Please see (top older, bottom newer):
https://software.intel.com/sites/landingpage/IntrinsicsGuide/